packages/ocvImpact/R/utils_montagu.R

```diff
@@ -53,6 +41,35 @@ import_coverage_scenario <- function(modelpath, country, scenario, num_doses = N
     if (filter0){
```

Comment:
This filtering would need to occur after adjust montagu coverage and apply only if coverage is 0 for both ocv1 and ocv2